examples/src/content/Draggable/DragEvents/index.js

```diff
@@ -61,9 +61,6 @@ export default function DragEvents() {
   });
 
   draggable.on('drag:move', (evt) => {
-    // Required to help restrict the draggable element to the container
-    evt.cancel();
```

Comment:
@tsov I don't know how this was working for so long... but `drag:move` is not cancellable... thing is, at one point this worked and it prevented the "Pill Switch" from being pulled outside of the "Track".

Is there a similar trick I can use here?

Reply:
Hmm, I will check this branch out later and see what's up, `drag:move` should be a cancelable event https://github.com/Shopify/draggable/blob/master/src/Draggable/Draggable.js#L492